CONTRIBUTING.md

```diff
@@ -1,62 +1,83 @@
 # Contributing to Fresco
+
 We want to make contributing to this project as easy and transparent as
 possible.
 
+## Security bugs
+
+Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe disclosure of security bugs. In those cases, please go through the process outlined on that page and do not file a GitHub issue.
+
+## Pull Requests
+
+We welcome pull requests.
+
+1. Fork the repo and create your branch from `master`. 
+2. If you've added code that should be tested, add tests.
+3. If you've changed APIs, update the documentation. 
+4. Make sure the test suite passes.
+5. Make sure your code passes lint.
+6. If you haven't already, complete the Contributor License Agreement ("CLA").
```

Comment:
Is there a link with more instructions on how to do this? If not maybe it's worth just adding a note that there will be an automated comment with more instructions.